fsnotify.go

```diff
@@ -0,0 +1,103 @@
+//+build fsnotify
```

Comment:
@commondream The default for this PR is actually to be without fsnotify. It has to be built with `go build -tags fsnotify` Though, it seems to have a conflict currently.